### ISSUE TITLE
Use different version numbers for Indigo and Jade

### ISSUE THREAD

**Issue by 130s** [moveit/moveit_setup_assistant#117]
tl;dr **(1)** Released version of this time Indigo is `0.7.2` and Jade is `0.7.1`, **(2)** At the next release for jade, it should be bumped, maybe to `0.8.x`.

I was about to make Indigo release but then faced a problem.

Since the branches for I and J are separated, which indicates that there can be difference between the two versions, we should use different version numbering scheme for them. But [Jade release was just made with `0.7.1`](https://github.com/ros/rosdistro/pull/11975) while the last release for Indigo is `0.7.0`. Now `catkin_prepare_release` doesn't move forward for me to bump a patch version for `indigo-devel`, since `0.7.1` tag that was created from `jade-devel` branch already exists. 

This particular behavior of catkin_prepare_release is I think correct. The problem, however, is what version I should use for the new version of Indigo.
- Not the best solution but I just bump indigo to `0.7.2`.
- [ ] TODO At the next release for jade, it should be bumped, maybe to `0.8.x`.

Sorry for the mess and confusion.

**Comment by mikeferguson** [comment]
This should have been resolved as part of moving to the new consolidated repo
